packages/ckeditor5-source-editing/docs/features/source-editing.md

```diff
@@ -39,7 +39,7 @@ The source editing plugin also works well with the {@link features/markdown Mark
 ## Installation
 
 <info-box>
-	Currently, the source editing mode is supported in the {@link examples/builds/classic-editor classic editor}. The source editing feature is not compatible with {@link features/collaboration CKEditor 5 collaboration features}. If you would like to use collaboration features, but for some reason you would like to also enable source editing, please [contact us](https://ckeditor.com/contact/).
+	Currently, the source editing mode is supported in the {@link examples/builds/classic-editor classic editor}. The source editing feature is not compatible with {@link features/real-time-collaboration CKEditor 5 real-time collaboration features}, but it can be used with {@link features/collaboration#non-real-time-collaboration non real-time collaboration}. If you would like to use collaboration features, but for some reason you would like to also enable source editing, please [contact us](https://ckeditor.com/contact/).
```

Comment:
```suggestion
	Currently, the source editing mode is supported in the {@link examples/builds/classic-editor classic editor}. The source editing feature can be used with {@link features/collaboration#non-real-time-collaboration collaboration features} except of {@link features/real-time-collaboration CKEditor 5 real-time collaboration}. If you would like to use the real-time collaboration mode, but for some reason, you would like to also enable source editing, please [contact us](https://ckeditor.com/contact/).
```

Let's inform that it works with collaboration in general and then let's inform that it doesn't work in real-time mode specifically.

Reply:
I've prepared a change very similar to your proposition within the same spirit. 
Could you take another look?